### PR TITLE
Split Data without using External Libraries #1

### PR DESCRIPTION
**Pull request by NeonKazuha**
Split data into X_train, X_test, y_train, y_test without using any external library such as train_test_split and numpy, instead used a function.
Screenshot 2024-03-15 123454

### CODE REVIEW

**Comment by NeonKazuha** [comment]
@darshbaxi @rootCircle Please review.

**Comment by darshbaxi** [comment]
@NeonKazuha I noticed that in the `train_test_split` function, the dataset splitting is deterministic rather than random. This means that the order of the data affects how it's split into training and testing sets. To ensure that the splits are more representative and to introduce randomness into the process, I recommend shuffling the data before splitting.

**Comment by darshbaxi** [comment]
@NeonKazuha I am closing this pull request since there are multiple pull requests from you addressing the same issue.
